db/seeds.rb

```diff
@@ -1,30 +1,54 @@
-require "factory_girl"
-require_relative "../spec/factories"
-require_relative "../spec/factories/users"
-require_relative "../spec/factories/websites"
-
-Group.create!([
-  {id: 1, title: "collection"},
-  {id: 2, title: "website"},
-  {id: 3, title: "exhibitions"},
-  {id: 4, title: "events "},
-])
-Metum.create!([
-  {id: 1, title: "Alt", instructions: "A long description is a lengthier text than a traditional alt-text that attempts to provide a . comprehensive representation of an image. Long descriptions can range from one sentence to several paragraphs."},
-  #{id: 2, title: "Caption", instructions: "This is the caption text to be displayed visibly. This text will also be read to a blind user so it is best not to repeat info from this item in the alt text."},
-  {id: 3, title: "Long", instructions: "A long description is a lengthier text than a traditional alt-text that attempts to provide a . comprehensive representation of an image. Long descriptions can range from one sentence to several paragraphs."}
-])
-Status.create!([
-  {id: 1, title: "Ready to review"},
-  {id: 2, title: "Approved"},
-  {id: 3, title: "Not approved"}
-])
-#User.create!([
-  #{id: 1, email: ENV["SUPPORT_EMAIL"], password: ENV["SUPPORT_PASSWORD"], first_name: "Support", last_name: "User", admin: true}
-#])
+# require 'factory_girl_rails'
+# require_relative "../spec/factories"
+# require_relative "../spec/factories/users"
+# require_relative "../spec/factories/websites"
+
+class Seed
+
+  def initialize
+    generate_groups
+    generate_metum
+    generate_status
+    generate_admin_user
+  end
```

Comment:
People using this class might not expect the initializer to have side-effects. Can you change this method to be  a separate `.generate` method? Also recommend adding a short comment above the class definition stating what the purpose of the class is.